src/view/UserphraseView.cpp

```diff
@@ -26,8 +26,8 @@ UserphraseView::UserphraseView(QWidget *parent)
     :QListView{parent}
     ,UserphraseDialog_{new UserphraseDialog{this}}
     ,menu_(new UserphraseViewMenu{this})
-    ,originalPhrase{new QString()}
-    ,originalBopomofo{new QString()}
+    ,originalphrase_(QString())
+    ,originalbopomofo_(QString())
```

Comment:
You mean the `originalphrase_` and `originalbopomofo_` don't need to initialize here?
So I just assign `originalphrase_ = userphrase->phrase_` and `originalbopomofo_ = userphrase->bopomofo_`?